README.md

```diff
@@ -109,5 +109,18 @@ The linter provides the possibility to implement these customized specifications
 
 More information about how this feature works is documented [here](docs/customize_specifications.md)
 
+## Pre-Commit
+If you'd like cfn-lint to be run automatically when making changes to files in your Git repository, you can install [pre-commit](https://pre-commit.com/) and add the following text to your repositories' `.pre-commit-config.yaml`:
+
+```yaml
+repos:
+-   repo: https://github.com/awslabs/cfn-python-lint
+    rev: v0.7.4 # The version of cfn-lint to use
+    hooks:
+    -   id: cfn-python-lint
+```
```

Comment:
I think you'll also want to mention:

```yaml
    files: path/to/cfn/dir/.*\.(json|yml|yaml)$
```

the default configuration will lint all yaml / json files in the repository (which probably (well definitely, since at least one is .pre-commit-config.yaml!) won't work).